backend/tilavarauspalvelu/api/rest/views.py

```diff
@@ -137,3 +144,47 @@ def readiness_check(request: WSGIRequest) -> JsonResponse:
         return JsonResponse({"status": "NOT_READY", "error": str(error)}, status=503)
 
     return JsonResponse({"status": "READY"}, status=200)
+
+
+@require_GET
+@csrf_exempt  # NOSONAR
+def reservation_unit_export(request: WSGIRequest) -> HttpResponse:
+    """Export reservation units to JSON."""
+    authorization = request.META.get("HTTP_AUTHORIZATION", "")
+    if authorization != settings.EXPORT_AUTHORIZATION_TOKEN:
+        msg = "Not authorized to export reservation units."
+        return HttpResponseForbidden(msg)
+
+    try:
+        reservation_units = [int(pk) for pk in request.GET.get("only", "").split(",") if pk]
+    except (ValueError, TypeError):
+        msg = "'only' should be a comma separated list of reservation unit ids."
+        return HttpResponseBadRequest(msg)
+
+    queryset = ReservationUnit.objects.all()
+    if reservation_units:
+        queryset = queryset.filter(pk__in=reservation_units)
+
+    exporter = ReservationUnitExporter(queryset=queryset)
+    data = exporter.write_json()
+
+    return JsonResponse(data, safe=False, status=200)
+
+
+@require_GET
+@csrf_exempt  # NOSONAR
+def reservation_statistics_export(request: WSGIRequest) -> HttpResponse:
+    """Export reservation statistics to JSON."""
+    authorization = request.META.get("HTTP_AUTHORIZATION", "")
+    if authorization != settings.EXPORT_AUTHORIZATION_TOKEN:
+        msg = "Not authorized to export reservation statistics."
+        return HttpResponseForbidden(msg)
+
+    queryset = ReservationStatistic.objects.all()
```

Comment:
I guess some kind of filter is needed but that could be added later after agreeing what it should be. (The data can be updated until the reservations are in the past, so the client can't just naively query today's statistics but maybe doesn't want a full update every time either. But if there is no need not know about upcoming reservations until they have taken place, a chunk containing the recent past could be queried on a weekly or daily basis.)

Reply:
We should ask Milla to find out what kind of filter they would like, if any.

Reply:
Asked Milla and she said I should ask Jesse, but since he's sick I'll add the filters later.